notebooks/270-sound-generation/270-sound-generation.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def _generate(prompt, negative_prompt, audio_length_in_s, num_inference_steps):
</pre><p>You can show progress bar in the Gradio interface by adding last parameter <code>_=gr.Progress(track_tqdm=True)</code> to the function and that requires <code>demo.queue().launch()</code> call  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>done</p>